_layouts/people.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
```

Comment:
```suggestion
layout: conference
```
This should be `conference`, to keep the side bar menu displayed.